package.json

```diff
@@ -1,29 +1,32 @@
 {
-	"name": "dojo-core",
-	"version": "2.0.0-pre",
-	"description": "Basic utilites for common TypeScript development",
-	"homepage": "http://dojotoolkit.org",
-	"bugs": {
-		"url": "https://github.com/dojo/core/issues"
-	},
-	"license": "BSD-3-Clause",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/dojo/core.git"
-	},
-	"devDependencies": {
-		"benchmark": "^1.0.0",
-		"dts-generator": "1.4.1",
-		"grunt": "0.4.5",
-		"grunt-contrib-clean": "0.6.0",
-		"grunt-contrib-copy": "0.8.0",
-		"grunt-contrib-watch": "0.6.1",
-		"grunt-text-replace": "0.4.0",
-		"grunt-ts": "3.0.0",
-		"grunt-tslint": "2.0.0",
-		"tslint": "2.1.1",
-		"intern": "2.2.2",
-		"requirejs": "2.1.17",
-		"typescript": "Arnavion/typescript-github#2015-05-07-e1d05150f6acef05452772192a4abdfc7eb671eb"
-	}
+  "name": "dojo-core",
+  "version": "2.0.0-pre",
+  "description": "Basic utilites for common TypeScript development",
+  "homepage": "http://dojotoolkit.org",
+  "bugs": {
+    "url": "https://github.com/dojo/core/issues"
+  },
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dojo/core.git"
+  },
+  "devDependencies": {
+    "benchmark": "^1.0.0",
+    "dts-generator": "1.4.1",
+    "grunt": "0.4.5",
+    "grunt-contrib-clean": "0.6.0",
+    "grunt-contrib-copy": "0.8.0",
+    "grunt-contrib-watch": "0.6.1",
+    "grunt-text-replace": "0.4.0",
+    "grunt-ts": "3.0.0",
+    "grunt-tslint": "2.0.0",
+    "tslint": "2.1.1",
+    "intern": "2.2.2",
+    "requirejs": "2.1.17",
+    "typescript": "Arnavion/typescript-github#2015-05-07-e1d05150f6acef05452772192a4abdfc7eb671eb"
+  },
+  "dependencies": {
+    "sinon": "~1.14.1"
```

Comment:
This should be in `devDependencies` and also use a hard version instead of a fuzzy one